data/human/deep jobs.txt

```diff
@@ -353,3 +353,616 @@ mission "Record academic conference"
+		not attributes "station" "moon"
+	on accept
+		dialog phrase "secure package pickup"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 80000 15000
+		"reputation: Deep Security" += 0.5
+		dialog phrase "secure package dropoff payment"
+	on fail
+		"reputation: Deep Security" -= 2
+	on abort
+		"reputation: Deep Security" -= 5
+
+
+mission "Deep Consultants"
+	minor
+	source
+		attributes "deep"
+		attributes "urban"
+		not attributes "station" "moon"
+		not planet "Valhalla"
+	destination "Valhalla"
+	to offer
+		"reputation: Deep Security" >= 100
+		"secure deep package" >= 30
+		"mystery cube" >= 5
+		has "deep security reveal"
+		or
+			not "chosen sides"
+			has "main plot completed"
+	passengers 2
+	deadline 2 1
+	on offer
+		conversation
+			`Strolling through the spaceport you are suddenly accosted by a young and very breathless woman. "Please, Captain," she wheezes, "I'm so glad I caught you."`
+			`	After drawing in a deep breath she stands up straighter and gestures to someone some distance away. "Captain, I'm sorry, but my colleague Julie and I are in dire need of urgent transport. Can you help?"`
+			`	As she catches her breath some more, her older colleague hustles over, bags in tow.`
+			choice
+				`	"Sure, I'd be glad to help.`
+					goto sure
+				`	"Where do you need to go?"`
+				`	"Why is it so urgent?"`
+			`	Julie holds out her hand in greeting. "Apologies, let's do this properly. My name is Julie Gillar and my assistant and I were held back with our duties and missed our booked transport. We're in dire need of returning to Valhalla for several important business meetings but unfortunately everything is booked up for the next five days straight, and that's just not acceptable to my employer."`
+			choice
+				`	"Sure, I'd be glad to help.`
+					goto sure
+				`	"Who's your employer?"`
+			`	Julie furtively glances at the people nearby. "Well, I have a few employers actually but at the moment, indirectly, Deep Security."`
+			`	She leans in slightly closer and lowers her voice. "We have access to their list of trustworthy pilots for emergencies. You'll be paid quite well."`
+			choice
+				`	"Sure, I'd be glad to help.`
+					goto sure
+				`	"I'm sorry, but I really can't help you right now."`
+					decline
+			label sure
+			`	"Lead right on then," says Julie.`
+				accept
+	on complete
+		log "Minor People" "Julie Gillar" `A consultant who worked with Deep Security. She requested emergency transport from you and showed you how to make yourself available to future consultants.`
+		payment 350000
+		set "deep security consultants"
+		conversation
+			`Julie and her assistant are model passengers the entire journey. As she prepares to disembark she thanks you for your assistance. In addition to transferring you <payment>, she points out how you can change your settings with the local spaceport agencies to make yourself available for business passengers consulting with Deep Security.`
```

Comment:
```suggestion
			`Julie and her assistant are model passengers the entire journey. As she prepares to disembark she thanks you for your assistance. In addition to transferring <payment> into your account, she helps you change your job board perferences with the local spaceport agencies to make yourself available for business passengers consulting with Deep Security.`
```

Reply:
'preferences' is misspelled, but otherwise fair

Reply:
derp

Reply:
feel free to mark as resolved if you've changed it without introducing that error. :)